spanner/pdml.go

```diff
@@ -51,14 +51,19 @@ func (c *Client) partitionedUpdate(ctx context.Context, statement Statement, opt
 	if err := checkNestedTxn(ctx); err != nil {
 		return 0, err
 	}
-
-	sh, err := c.idleSessions.take(ctx)
+	var sh *sessionHandle
+	if c.idleSessions.isMultiplexedSessionForPartitionedOpsEnabled() {
+		sh, err = c.idleSessions.takeMultiplexed(ctx)
+	} else {
+		sh, err = c.idleSessions.take(ctx)
+	}
 	if err != nil {
 		return 0, ToSpannerError(err)
 	}
-	if sh != nil {
-		defer sh.recycle()
+	if sh == nil {
+		return 0, spannerErrorf(codes.Internal, "no session available")
 	}
+	defer sh.recycle()
```

Comment:
What does this mean for multiplexed session?

Reply:
It will decrement the multiplex session usage metric https://github.com/googleapis/google-cloud-go/blob/main/spanner/session.go#L389